application/views/template_view.py

```diff
@@ -0,0 +1,20 @@
+from flask import Blueprint, make_response, request, jsonify
+from application.controllers.template_controller import TemplateController
+
+template_view = Blueprint('template_view', __name__)
+
+
+@template_view.route('/template/<template_id>', methods=['POST'])
+def upload_template(template_id):
+    template = request.get_json()
+    msg = TemplateController.upload_comms_template(template_id, template)
+    return make_response(msg, 201)
```

Comment:
We don't need to send a message back in this case because 201 by definition means that the resource has been created